docs/conf/portland/2020/sponsors/online-prospectus.rst

```diff
@@ -0,0 +1,171 @@
+#############################
+
+.. contents:: Sections
+   :local:
+   :depth: 1
+   :backlinks: none
+
+Introduction
+============
+
+Welcome to our brand new **Online Conference Sponsorship Prospectus**.
+This document has been updated from the original :doc:`prospectus` for our Portland 2020 conference,
+including new benefits and expanded value across our community.
+
+We're excited to work with the organizations in our community to build the best possible event in 2020.
+In particular, we would love your feedback on sponsorship levels and benefits.
+We've expanded the offerings and have introduced a lot of new things,
+but please let us know if there are other points of interaction with our community that would be valuable for you.
+
+Concept
+=======
+
+{% include "conf/sponsorship/concept.rst" %}
+
+Demographics
+============
+
+{% include "conf/sponsorship/demographics.rst" %}
+
+Why Sponsor
+===========
+
+{% include "conf/sponsorship/why.rst" %}
+
+Sponsorship Packages
+====================
+
+The following options are suggested sponsorship levels. We are happy to discuss adjustments and custom packages.
+
+.. warning:: This is a draft prospectus and is subject to change.
+
+Second Draft
+------------
+
+The **Second Draft** package is great for companies looking to hire or promote a product.
+
+- **Increased** Three (3) tickets_
+- **Increased** Two (2) featured job postings on our Job Board, also promoted in our newsletter ({{ newsletter_subs }} subscribers)
+- **Increased** A short description and logo (200 words) of your company on the conference website
+- A table at our **virtual job fair**
+- Name included in welcome announcement in email newsletters and social media
+
+The **Second Draft** package costs **{{sponsorship.second_draft.price}}**.
+
+Publisher
+---------
+
+The **Publisher** package is great for sending a team and getting to know the community.
+
+- **Increased** Seven (7) tickets_
+- **Increased** Four (4) featured job postings on our Job Board, also promoted in our newsletter ({{ newsletter_subs }} subscribers)
+- **Increased** A short description and logo (200 words) of your company on the conference website
+- A table at our **virtual job fair**
+- Name included in welcome announcement in email newsletters and social media
+- **New** A small logo on all Write the Docs website pages for 3 months
+
+The **Publisher** package costs **{{sponsorship.publisher.price}}**.
+
+Patron
+------
+
+**Limit 3**
+
+The **Patron** package highlights your company as a force in the industry and community:
+
+- **Increased** Thirteen (13) tickets_
+- **Increased** Seven (7) featured job postings on our Job Board, also promoted in our newsletter ({{ newsletter_subs }} subscribers)
+- **Increased** A medium description and logo (500 words) of your company on the conference website
+- A **virtual sponsorship booth**
+- A featured table at our **virtual job fair**
+- Small logo included in intermission slides and on talk videos
```

Comment:
Not sure about the "on talk videos" here, do you mean like a watermark on the corner of the video? I think intermission slides is enough, or maybe on the closing slide that we can fade to after the video is finished. Feels wrong to force continuous sponsorship logos in the talk videos themselves...

Reply:
This is the same as previous years, where we have the logo slide on the talk videos with sponsor logos.